backend/docker/entrypoint.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+if [ "$DATABASE" = "postgres" ]
+then
+    echo "Waiting for postgres..."
+
+    while ! nc -z $SQL_HOST $SQL_PORT; do
+      sleep 0.1
+    done
+
+    echo "PostgreSQL started"
+fi
+
+# python manage.py flush --no-input
+pip install -r requirements-dev.txt
```

Comment:
Is er een requirements-dev? anders deze weglaten en gewoon requirements.txt toevoegen om verwarring te voorkomen

Reply:
Ja, die is er. Staat naast requirements.txt. Reden van de -dev is dat je bijvoorbeeld packages als 'black' niet op de server wilt hebben. 
Straks bij deploy moeten we gebruik maken van entrypoint.prod.sh in plaats van entrypoint.sh.